### PR TITLE
Calls -> Voice Renaming

### PR DESCRIPTION
**Pull request by hummusonrails**
Related to #151 

Renames `Calls` to `Voice`

### CODE REVIEW

File: sorbet/rbi/todo.rbi [hunk 1]
```diff
@@ -10,17 +10,7 @@ module Nexmo::Applications::Basic; end
...
-module Nexmo::Calls::DTMF::Put; end
-module Nexmo::Calls::Post; end
-module Nexmo::Calls::Put; end
-module Nexmo::Calls::Stream::BearerToken; end
-module Nexmo::Calls::Stream::Delete; end
-module Nexmo::Calls::Stream::Put; end
-module Nexmo::Calls::Talk::BearerToken; end
-module Nexmo::Calls::Talk::Delete; end
-module Nexmo::Calls::Talk::Put; end
+module Nexmo::Calls; end
```
Comment (kellyjandrews) [inline sorbet/rbi/todo.rbi]: If we are removing `Calls`, what does this do?

**Review by kellyjandrews — APPROVED** [review]

**Comment by AlexLakatos** [comment]
This will be a breaking change. Is there any value in keeping both for this version, and adding a deprecation message, with a plan to remove Calls further down the release cycle?

**Comment by hummusonrails** [comment]
> This will be a breaking change. Is there any value in keeping both for this version, and adding a deprecation message, with a plan to remove Calls further down the release cycle?

@timcraft and I spoke at length about that, and we decided to use the opportunity of this major version release, which adds another breaking change, to also introduce this one. Users usually look and expect to find breaking changes in a major version release, so it's a pretty safe move.

**Review by AlexLakatos — APPROVED** [review]
LGTM 👍

**Review by AlexLakatos — COMMENTED** [review]
It looks like `module Nexmo::Entity::Keys; end` is being removed and `module T::Private::Methods::MethodHooks; end` & `module T::Private::Methods::SingletonMethodHooks; end` are being added in the todo sorbet, and I can't figure out where that change is coming from.

**Comment by hummusonrails** [comment]
> It looks like `module Nexmo::Entity::Keys; end` is being removed and `module T::Private::Methods::MethodHooks; end` & `module T::Private::Methods::SingletonMethodHooks; end` are being added in the todo sorbet, and I can't figure out where that change is coming from.

It makes sense that `Nexmo::Entity::Keys` would be removed from that list, as `Keys` is discoverable. As to those two additional lines, I'm also not sure why they are there. It doesn't impact the type checking nor the performance whether they are are there or not, but I did experiment and ran and re-ran the todo regenerator and each time they were added in that regeneration.

**Comment by hummusonrails** [comment]
Noticed that the newest releases of Sorbet removed duplications in the generator, which when used cleaned up this file further. Note -- this is not directly related to the Voice renaming, and doesn't impact it, but is tangentially connected.

**Pull request merged**